### PR TITLE
Add documentation and examples for `NonUniformStaticScale`.  Remove the no-argument constructor and make the dictionary constructor explicit

### PR DESCRIPTION
**Pull request by alexanderbock**

### CODE REVIEW

**Review by engbergandreas — APPROVED** [review]

**Review by WeirdRubberDuck — CHANGES_REQUESTED** [review]
One comment about the usage of the word "simple", and a very long description

File: data/assets/examples/scale/nonuniformstaticscale/simple.asset [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+}
+
+asset.onInitialize(function()
+  openspace.addSceneGraphNode(Node)
+end)
+
+asset.onDeinitialize(function()
+  openspace.removeSceneGraphNode(Node)
+end)
```
[the remark this answers is not in the document]
Comment (engbergandreas) [reply data/assets/examples/scale/nonuniformstaticscale/simple.asset:31]: We decided on not adding meta and export, from Slack thread: 
"Don't add: meta, export,
Add: Gui tag, initialize, deinitialize" - Ylva

[the remark this answers is not in the document]
Comment (WeirdRubberDuck) [reply data/assets/examples/scale/nonuniformstaticscale/simple.asset:31]: Thanks for clarifying.  There were too many discussions goin on simultaneously during that session for me to be able to follow along 😓

File: data/assets/examples/scale/nonuniformstaticscale/ellipsoid.asset [hunk 2]
```diff
@@ -0,0 +1,29 @@
+-- This asset creates a SceneGraphNode that is rendering a sphere which is adjust to an
+-- ellipsoidal shape by using a non-uniform scaling. In particular, the second axis of
+-- symmetry is half as long as the first, and the third axis of symmetry is a third as
```
Comment (ylvaselling) [inline data/assets/examples/scale/nonuniformstaticscale/ellipsoid.asset:3]: I'm a little confused what the axes of symmetries are - is there a way to rephrase this? First, second, and third axis? x, y, and z?

File: data/assets/examples/scale/nonuniformstaticscale/simple.asset [hunk 3]
```diff
@@ -0,0 +1,31 @@
+-- This asset creates a SceneGraphNode that only displays coordinate axes. The coordinate
+-- axis normally have a length of 1 meter and are scaled in this example by a factor of
+-- 149597870700 along the x axis, which means they will be 149597870700 m (1 AU) long,
+-- thus reaching the same distance as Earth's orbit around the Sun, while the y-axis stays
+-- at its original size, and the z-axis will be removed entirely by setting the scale
+-- value to 0.
```
Comment (ylvaselling) [inline data/assets/examples/scale/nonuniformstaticscale/simple.asset:6]: Maybe put in a comment as to why the value isn't actually 0 below but 0.0005.

File: modules/base/scale/nonuniformstaticscale.cpp [hunk 4]
```diff
@@ -36,6 +36,16 @@ namespace {
         openspace::properties::Property::Visibility::NoviceUser
     };
 
+    // This Scale type scales the scene graph node that it is attached to by a fixed
+    // amount that does not change over time. It is possible to change the fixed scale
+    // after starting the application, but it otherwise remains unchanged. The scaling is
+    // a simple multiplication so that a `Scale` value of 10 means that the object will be
+    // 10 times larger than it should be. In comparison to the StaticScale type, this type
```
Comment (ylvaselling) [inline modules/base/scale/nonuniformstaticscale.cpp:43]: Maybe rephrase "what it should be" to "original size"

File: modules/base/scale/nonuniformstaticscale.cpp [hunk 5]
```diff
@@ -36,6 +36,16 @@ namespace {
...
+    // has the ability to scale an object by different amounts for each direction.
+    //
+    // This type can be used to adjust the aspect ratio of certain Renderable types, for
```
Comment (ylvaselling) [inline modules/base/scale/nonuniformstaticscale.cpp:46]: Only certain renderable types? It isn't all?

File: modules/base/scale/nonuniformstaticscale.cpp [hunk 6]
```diff
@@ -36,6 +36,16 @@ namespace {
...
+    // example to make a RenderableSphericalGrid that is not a perfect spherical grid, but
+    // a tri-axial ellipsoid instead.
     struct [[codegen::Dictionary(NonUniformStaticScale)]] Parameters {
         // [[codegen::verbatim(ScaleInfo.description)]]
         glm::dvec3 scale;
```
Comment (ylvaselling) [inline modules/base/scale/nonuniformstaticscale.cpp:51]: Is this a positive value only?

File: data/assets/examples/scale/nonuniformstaticscale/ellipsoid.asset (hunk 2 above)
Comment (alexanderbock) [reply data/assets/examples/scale/nonuniformstaticscale/ellipsoid.asset:3]: Fine to rename it to just "axes", but from Wikipedia:
> An ellipsoid has three pairwise [perpendicular](https://en.wikipedia.org/wiki/Perpendicular) [axes of symmetry](https://en.wikipedia.org/wiki/Rotational_symmetry) which intersect at a [center of symmetry](https://en.wikipedia.org/wiki/Central_symmetry), called the center of the ellipsoid

File: modules/base/scale/nonuniformstaticscale.cpp (hunk 6 above)
Comment (alexanderbock) [reply modules/base/scale/nonuniformstaticscale.cpp:51]: There might be reasons to have a negative scale, in case someone wants to mirror a model for example.

**Review by Roxeena — COMMENTED** [review]
:+1:
